app/importers/market_research_data.rb

```diff
@@ -3,9 +3,9 @@
 
 class MarketResearchData
   include Importable
-  include VersionableResource
```

Comment:
You shouldn't need to remove this, and it's better if you can keep it in place TBH. Since you have two sources, you need to override `available_version` so that it returns a meaningful value. Take a look at [SBA Trade Events](https://github.com/GovWizely/webservices/blob/master/app/importers/trade_event/sba_data.rb#L24), which is similar to what you want to do.